src/draw/lv_draw_buf.h

```diff
@@ -24,6 +25,12 @@ extern "C" {
  *      TYPEDEFS
  **********************/
 
+typedef struct {
+    lv_image_header_t header;
+    uint32_t data_size;     /*Total buf size in bytes*/
```

Comment:
Why do we need this?

Reply:
`data_size` is important when we using the buffer.
E.g. save an image to file system, make an copy of `draw buf`. 
Or, it can be used as double-check before accessing memory.
Or as a debug measure to check crash reasons.